cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -215,7 +220,9 @@ if(builtin_lzma)
       CONFIGURE_COMMAND ""
       BUILD_COMMAND ""
       INSTALL_COMMAND ""
-      LOG_DOWNLOAD 1)
+      LOG_DOWNLOAD 1
+      TIMEOUT ${timeoutProject}
```

Comment:
We only need the `TIMEOUT` parameter for externals that we download; this one is picked up from within the source tree. IIUC `TIMEOUT` has no effect here. Please remove for all local URLs

Reply:
@Axel-Naumann Thank you for the suggestions. 
Imm make the required changes.
I'll need to remove the parameters only in this file right?
https://github.com/root-project/root/blob/db09e093521829f1cb6aac7af194f681b1f19548/cmake/modules/SearchInstalledSoftware.cmake